pkg/cli/build/build.go

```diff
@@ -199,6 +201,31 @@ func isImageDefinitionValid(ctx *image.Context) bool {
 	return true
 }
 
+func appendElementalRPMs(ctx *image.Context) {
+	elementalDir := filepath.Join(ctx.ImageConfigDir, "elemental")
+	if _, err := os.Stat(elementalDir); err != nil {
+		if !errors.Is(err, fs.ErrNotExist) {
+			zap.S().Warnf("Looking for 'elemental' dir failed unexpectedly: %s", err)
+		}
+
+		return
+	}
+
+	audit.Audit("Elemental registration is configured. The necessary RPM packages will be downloaded.")
+	zap.S().Info("Elemental registration requested")
```

Comment:
`AuditInfo` will do both write to the console and log the message; is there a reason the messages need to be different?

Reply:
As mentioned on the other PR, not a hard "requirement" but I definitely don't see the point of being verbose / user friendly in the log files. I tend to prefer splitting up what the user sees from what the log files contain even at the cost of duplicating code.

I can change if you favour the `AuditInfo` approach, though.